src/component/services.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+const Container = Styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const WContainer = Styled.div`
```

Comment:
`WContainer`, `EContainer`, `DContainer`, `OContainer`... not good names! What are they?